kubernetes_platform/proto/kubernetes_executor_config.proto

```diff
@@ -154,3 +155,11 @@ message ImagePullSecret {
     // Name of the image pull secret.
     string secret_name = 1;
 }
+
+message Toleration {
+    string key = 1;
+    string operator = 2;
+    string value = 3;
+    string effect = 4;

```

Comment:
Should this be an `enum`? No strong preference from me -- just want to mention. The enum adds better validation at the cost of more maintenance effort.

Reply:
Good question. Interestingly, the K8s protobuf tolerations message ([link](https://github.com/kubernetes/api/blob/master/core/v1/generated.proto#L5922-L5927)), which this message is modeled after, does **not** use `enum`. I think it might be a little strange if we hold our message to a higher standard of validation than their's, especially given that the K8s API handles validation for these objects on the backend. All that being said, happy to implement as `enum` if you think that makes more sense.

Reply:
SGTM! Thanks for looking into it.